requirements/prod-requirements.in

```
@@ -2,7 +2,8 @@
 
 pip
 
-flower
+# Switch back to normal package as soon as flower realease a version > 1.2.0
+flower @ git+https://github.com/mher/flower
```

I recommend pinning a specific ref. A tag would be better, but they haven't pushed one yet, so I've used the SHA of your PR's merge commit.

```suggestion
flower @ git+https://github.com/mher/flower@e06f2750
```